FlowCrypt/Functionality/Services/Compose Message Service/ComposeMessageService.swift

```diff
@@ -83,6 +82,15 @@ final class ComposeMessageService {
             return .failure(.validationError(.missedPublicKey))
         }
 
+        let sendableAttachments: [SendableMsg.Attachment] = contextToSend.attachments
```

Comment:
Please help me ensure that combined size of all attachment is under 10mb. If oversize, return a validation error informing the user (eventually we may implement streaming encryption and upload where the size would be unlimited as far as iOS is concerned, but we're not there yet)